brainy-ideas/readAloud-valence-jess/readme.md

```diff
@@ -0,0 +1,145 @@
+
+**Design:**
+
+Prior to the experimental task, participants complete several questionnaires:
+- Demographic questionnaire
+- Assessment of reading skill (TBD)
+- Brief Mood Introspection Scale (BMIS)
+- Emotion Regulation Questionnaire (ERQ)
+- Interpersonal Emotion Regulation Questionnaire (IERQ)
+- Beck Depression Inventory (BDI-II)
+- State-Trait Anxiety Inventory (STAI)
+- Affective Reactivity Index (ARI)
+- Interpersonal Reactivity Index (IRI)
+
+
+After completion of the questionnaires, participants are prompted to speak briefly (~1 minute) on any topic; this brief narrative is recorded.  Ostensibly, the purpose of this is to test and calibrate the online recording system.  For the initial analyses, this brief narrative does not serve any specific purpose; however, future analyses may be able to correlate qualities of the narrative with the BMIS measurement.
+
+The main study task involves the reading aloud of 21 short texts (~200 words each) that are designed to evoke a certain mood state in the reader (positive/negative) at the start of the passage, but include a single valence "switch point" where the mood of the text suddenly reverses. (Examples: [pos>neg passage](https://github.com/NDCLab/brainBox/blob/rwe-valence-jess/brainy-ideas/rwe-valence-jess/stimuli/pos-neg_dolphins.txt) and [neg>pos passage](https://github.com/NDCLab/brainBox/blob/rwe-valence-jess/brainy-ideas/rwe-valence-jess/stimuli/neg_pos_dolphins.txt).)  Seven blocks of texts, with each block including three texts, are presented back-to-back in such a way that "switch points" are passage-internal (not between passages) and the total amount of "positive" text is equal to the total amount of "negative" text.  Participants are instructed to read the texts aloud at a comfortable speed, articulating clearly and naturally as if they were rehearsing for a documentary voiceover.  Between each block, a "break" is provided during which the participant is prompted to speak on any topic (similar to the initially recorded narrative) and in keeping with the ecological validity of the study: people typically "chat" when they take breaks between regular life tasks.  The primary functions of these breaks are to give the participant a rest period from the tasking activity of reading aloud and to put the participant back in control of their own mood state before the next block.  A secondary function, if future analyses are able to correlate narrative qualities with the initial BMIS measurement, is to offer the possibility of identifying more dynamic changes in mood during the reading aloud task.
+
+After all seven blocks of the reading aloud task are complete, the participant is prompted to share their overall impression of the activity (~1 minute), which is recorded.  They then re-complete the BMIS.
+
+I believe that, with careful setup, this study could be completely asynchronous as [PsychoJS now supports microphone input](https://www.psychopy.org/changelog.html).
```

Comment:
the whole study? I think i misunderstand, as you were also proposing chatting in between blocks.

Reply:
Haha, yes, the participant was supposed to just chat with their screen.  As in, the screen gave them a prompt and they could just speak, randomly, for a minute or so.  But I'm fine to do something different as a "break," just need some guidance!

Reply:
Modified breaks.